terraform/modules/cloudfront/cloudfront.tf

```diff
@@ -122,6 +122,52 @@ resource "aws_cloudfront_distribution" "default" {
     viewer_protocol_policy = "redirect-to-https"
   }
 
+  cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = "${var.project_name}-${var.environment}-default-origin"
+
+    path_pattern = "/"
+
+    forwarded_values = {
+      query_string = false
+      headers      = ["Host", "Authorization"]
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    min_ttl     = 300 # 5 minutes
+    default_ttl = 900 # 15 minutes
+    max_ttl     = 3600 # 60 minutes
+
+    viewer_protocol_policy = "redirect-to-https"
+  }
+
+  cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = "${var.project_name}-${var.environment}-default-origin"
+
+    path_pattern = "/jobs"
+
+    forwarded_values = {
+      query_string = true
+      headers      = ["Host", "Authorization"]
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    min_ttl     = 60 # 1 minute
+    default_ttl = 60 # 1 minute
```

Comment:
This will stop a proportion of requests making it to our origin so Rails won't be able to record searches that are cached. I propose that the DOS reliability is worth the trade for losing a smaller percentage of audits for search data.